### PR TITLE
Feat/bnbx support

### PR DESCRIPTION
**Pull request by carlomazzaferro**
## Type of change

- [ ] Docs change / dependency upgrade
- [ ] Configuration / tooling changes
- [ ] Refactoring
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires changes in customer code

## Related Issue(s)

Fixes

### CODE REVIEW

File: packages/chains/src/bsc/assets.ts [hunk 1]
```diff
@@ -1,8 +1,18 @@
...
+  BNBxDocs,
+  defaultDocs,
+  ellipsisDocs,
+  jarvisDocs,
+  pancakeSwapDocs,
+  stkBNBDocs,
+} from "../common";
 
 export const WBNB = "0xbb4CdB9CBd36B01bD1cBaEBF2De08d9173bc095c";
+export const BNB = "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE";
```
Comment (peetzweg) [inline packages/chains/src/bsc/assets.ts:15]: lol? Is it a real token? Looks like an EOA to me.
So native token? Maybe use ZeroAddress again?

File: packages/chains/src/bsc/assets.ts [hunk 2]
```diff
@@ -27,6 +37,9 @@ const xBOMB = "0xAf16cB45B8149DA403AF41C63AbFEBFbcd16264b";
 const aBNBc = "0xE85aFCcDaFBE7F2B096f268e31ccE3da8dA2990A";
 const stkBNB_WBNB = "0xaA2527ff1893e0D40d4a454623d362B79E8bb7F1";
 const stkBNB = "0xc2E9d07F66A89c44062459A47a0D2Dc038E4fb16";
+const asBNBx_WBNB = "0xB88F211EC9ecfc2931Ae1DE53ea28Da76B9Ed37A";
```
Comment (peetzweg) [inline packages/chains/src/bsc/assets.ts:40]: Ape Swap LP is correct?

**Review by peetzweg — APPROVED** [review]
lgtm

File: packages/chains/src/bsc/assets.ts (hunk 1 above)
Comment (carlomazzaferro) [reply packages/chains/src/bsc/assets.ts:15]: That's the address for native BNB on BSC. See: https://ellipsis.finance/pool/0xFD4afeAc39DA03a05f61844095A75c4fB7D766DA check the BNB token. It is the null address

File: packages/chains/src/bsc/assets.ts (hunk 2 above)
Comment (carlomazzaferro) [reply packages/chains/src/bsc/assets.ts:40]: Yes, BNBx is on ApeSwap

**Pull request merged**